Pipelines/Config/settings.yaml

```diff
@@ -0,0 +1,8 @@
+variables:
+  # Note that when updating this value also ensure that you update the
+  # following locations to match. There will be a CI failure if they don't
+  # match (see scripts/packaging/versionmetadata.ps1)
+  # ProjectSettings/ProjectSettings.asset:  bundleVersion: x.x.x
+  # ProjectSettings/ProjectSettings.asset:  metroPackageVersion: x.x.x.0
+  MRTKGraphicsToolsVersion: 0.4.13
```

Comment:
We use this pattern in MRTK where we're patching multiple packages at once, but I'm wondering if it's necessary in this project where the version appears self-managed in the package.json 

https://github.com/microsoft/MixedReality-GraphicsTools-Unity/blob/29415a93c9145f2f7232bba28bc70df76ef67a6e/com.microsoft.mrtk.graphicstools.unity/package.json#L3

Reply:
I don't think it is necessary, however, I couldn't figure out how to consume the version from package.json in the yaml, that's why I left it in place..